contributor_docs/documentation_style_guide.md

```diff
@@ -0,0 +1,1184 @@
+
+Spell names correctly: p5.js, CSS, HTML, JavaScript, WebGL. When in doubt, refer to an authoritative source like their official documentation.
+
+**[⬆ back to top](#table-of-contents)**
+
+## Unbiased Documentation
+
+Write documentation without bias towards any kind of person. While documenting particularly demanding/sensitive topics, take the time to educate yourself. Ensure that your writing doesn’t hurt or offend someone unintentionally.
+
+While writing unbiased documentation:
+
+- Be inclusive of every gender identity and expression, sexual orientation, race, ethnicity, language, neuro-type, size, disability, class, religion, culture, subculture, political opinion, age, skill level, occupation, and background. Make examples as diverse as our community.
+- Avoid politicized content. If political content is necessary, remain neutral.
+- Follow accessibility guidelines.
+- Avoid content that would insult or cause harm to people.
+- Don’t make any generalizations about people, countries, and cultures. That includes positive or neutral generalizations.
+- Don’t write prejudiced and discriminatory content against minority communities.
+- Avoid terms related to historical events.
+
+Prefer wordings that avoid "you"s and "your"s. For example, instead of:
+
+```
+If you need to declare a variable, it is recommended that you use `const` or `let`.
+```
+
+use this style:
+
+```
+Use `const` or `let` to declare variables.
+```
+
+**Pronouns**
+
+| Recommended |	Not Recommended |
+| -- | -- |
+| they | he or she |
+| them | him or her |
+| their | his or her |
+| theirs | his or hers |
+| themselves | himself or herself |
+
+**[⬆ back to top](#table-of-contents)**
+
+## Accessibility and Disability
+
+- Emphasize the reader rather than underlining their inconveniences.
+- Don’t refer to a person with a disability as a disabled person. Use approved terminology for people with specific disabilities.
+- Maintain a uniform structure throughout the p5.js documentation. Emphasize important points both stylistically and visually.
+- Use a screen reader to test documentation. To test a screen reader, see [List of screen readers](https://en.wikipedia.org/wiki/List_of_screen_readers).
+- Consider multi-platform accessibility for all types of devices and operating systems.
+- Create examples with all types of input devices such as voice and gesture based devices, controllers, mice, and keyboards.
+- Don’t use ableist language. Be inclusive and unbiased while writing about accessibility and disability.
+- Take a pragmatic approach to HTML semantics. Don’t add semantics purely for the sake of semantics. If there is an HTML structure that clearly matches the content, use that element. For example, a group of links should most likely use a list element.
+- Use simple tables and tabular formats. Avoid span tags (such as rowspan and colspan). Tables prove to be difficult for screen readers.
+
+**Accessibility terminology**
```

Comment:
Hi, thanks for including the accessibility terminology. Did you refer to any specific resources while coming up with these terminologies? If so, could you include the links as reference?

Reply:
I drew these from the WordPress documentation guidelines for [Writing inclusive documentation](https://make.wordpress.org/docs/style-guide/general-guidelines/inclusivity/). The main page link is at the top of the style guide. Happy to reference the [specific section](https://make.wordpress.org/docs/style-guide/general-guidelines/inclusivity/#accessibility-terminology) in context.